stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -1123,6 +1129,20 @@ public func _TFCTerminateTensorComputation(_ computation: _TensorComputation) {
   computation.terminate()
 }
 
+/// Registers all functions in a graph into the eager context if in eager mode.
+///
+/// - Parameters:
+///   - programByteAddress: The address of the raw program.
+///   - programByteCount: The number of bytes in the program.
+@inlinable
+@_silgen_name("_swift_tfc_RegisterTensorFunctions")
```

Comment:
This should be a C convention function because all arguments are C-representable. With C-convention, it does not need serialization (`@inlinable`), and can be `internal`.

Concretely, this function signature should be:
```swift
@usableFromInline
@_cdecl("_swift_tfc_RegisterTensorFunctions")
func _TFCRegisterTensorFunctions(
  _ programByteAddress: UnsafeRawPointer,
  _ programByteCount: Int
)
```

Reply:
Acknowledged. Going to leave this way.